packages/react/package.json

```diff
@@ -283,5 +282,9 @@
     "whatwg-fetch": "^3.0.0"
   },
   "sideEffects": false,
-  "version": "2.146.0-next.33"
+  "version": "2.146.0-next.33",
+  "resolutions": {
```

Comment:
Yeah, technically they aren't doing anything b/c yarn 1 ignores package level resolutions in a monorepo setup, but @davidicus and I discussed adding them here yesterday just for clarity since they're actually being enforced by the root-level package.json.